dali/operators/operators.cc

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/core/api_helper.h"
+#include "dali/operators/operators.h"
+
+namespace dali {
+DLL_PUBLIC void InitOperatorsLib() {}
```

Comment:
You can add some comment why empty function does it job.

Reply:
The comment is in the header.